Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.yml

```diff
@@ -1735,9 +1735,20 @@ script:
       name: hostname
       required: false
       secret: false
+    - auto: PREDEFINED
+      default: false
+      defaultValue: 'false'
+      description: Whether to scan all of the endpoints or not. Default is false.
```

Comment:
mention that passing all=true could cause performance issues, because it will run scan on all the endpoints